.python-version

```diff
@@ -1 +1 @@
-3.11.10
```

Comment:
Not sure if we need to change this version. There was an issue during setup but do not remember if this change fixed the issue.